sonar-html-plugin/src/test/resources/checks/IndistinguishableSimilarElementsCheck/MultipleNavAside.html

```diff
@@ -0,0 +1,60 @@
+<nav> <!-- Noncompliant -->
+</nav>
+
+<nav> <!-- Noncompliant -->
+</nav>
+
+<nav aria-label="foo"> <!-- Compliant -->
+</nav>
+
+<nav aria-labelledby="foo">  <!-- Compliant -->
+</nav>
+
+<aside> <!-- Noncompliant -->
+</aside>
+
+<aside> <!-- Noncompliant -->
+</aside>
+
+<aside aria-label="foo"> <!-- Compliant -->
+</aside>
+
+<aside aria-labelledby="foo"> <!-- Compliant -->
+</aside>
+
+<div role="banner"> <!-- Noncompliant -->
+</div>
+
+<p role="banner">   <!-- Noncompliant -->
+</p>
+
+<strong role="Banner"> <!-- Noncompliant -->
+</strong>
+
+<table role="complementary"> <!-- Compliant -->
+</table>
+
+<div role="contentinfo"> <!-- Compliant -->
+</div>
+
+<div role="form"> <!-- Compliant -->
+</div>
+
+<div role="main"> <!-- Compliant -->
+</div>
+
+<div role="navigation"> <!-- Compliant -->
+</div>
+
+<div role="search"> <!-- Compliant -->
+</div>
+
+<div role="application"> <!-- Compliant -->
```

Comment:
I would have expected this one to be non-compliant since there is another element with role "application" and this div element does not have an aria label property